package.json

```diff
@@ -3,9 +3,6 @@
   "version": "1.0.0",
   "repository": "https://github.com/wagtail/wagtail",
   "private": true,
-  "engines": {
-    "node": ">= 4"
-  },
```

Comment:
I believe we could get around this by including an `.npmrc` in the repo root with `engine-strict = true`. `engineStrict` was removed as an option in `package.json` but still exists in npm settings.